client/src/app/core/repositories/motions/motion-block-repository.service.ts

```diff
@@ -129,12 +129,13 @@ export class MotionBlockRepositoryService extends BaseRepository<ViewMotionBlock
     }
 
     /**
-     * Retrieves motion block(s) by name
-     * TODO: check if a title is unique for a motionBlock
-     * @param title Strign to check for
+     * Retrieves motion blocks by name
+     *
+     * @param title String to check for
+     * @returns the motion blocks found
      */
-    public getMotionBlockByTitle(title: string): MotionBlock {
-        return this.DS.find(MotionBlock, block => block.title === title);
+    public getMotionBlocksByTitle(title: string): MotionBlock[] {
+        return this.DS.filter(MotionBlock, block => block.title === title);
     }
```

Comment:
MotionBlocks are currently not unique server side. This makes it clear that this is the case, and avoids confusion. It would be better to ensure motion block uniqueness somehow